content/tutorials/encryption-realtime.textile

```diff
@@ -0,0 +1,618 @@
+---
+title: Encryption Tutorial
+section: tutorials
+index: 21
+jump_to:
+ Section:
+   - 1. Introduction#intro
+   - 2. Step 2 - Install Ably#install-ably
+   - 3. Step 3 - Publish encrypted messages to a channel#publish-messages
+   - 4. Step 4 - Subscribe and decrypt messages#subscribe-messages
+   - 5. Live Demo#live-demo
+   - 6. Download tutorial source code#download-tutorial
+   - 7. Next Steps#next-steps
+
+---
+
+h2(#intro). Introduction
+
+Ably's "encryption feature":/realtime/encryption enables you to send encrypted messages on channels that can later be retrieved and decrypted using a common cryptographic key known only to the client and server. This has the advantage that Ably demonstrably has no access to the un-encrypted contents of your messages, but also means that each app is responsible for enabling the distribution of keys to clients independently of Ably.
+
+Using our "encryption API":/realtime/encryption is easy.  Let's get started.
+
+<%= partial 'tutorials/_step-1-setup-free-account' %>
+
+blang[java].
```

Comment:
This link doesn't make any sense. The GitHub step linked does not show any retrieving Ably API key as described in the tutorial step. Please remove it as it's not relevant.